docs/content/comparisons/druid-vs-computing-frameworks.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: doc_page
+---
+
+Druid vs General Computing Frameworks (Spark/Hadoop)
+====================================================
+
+Druid is much more complementary to general computing frameworks than it is competitive. General compute engines can 
+very flexibly process and transform data. Druid acts as a query accelerator to these systems by indexing raw data into a custom 
+column format to speed up OLAP queries.
+
+## Druid vs Hadoop
```

Comment:
I think this section is a bit vague - what is "Hadoop" in this context? Are you talking about Hive on HDFS?

I would guess that Impala running on HDFS with Parquet storage would likely be pretty competitive with Druid for a lot of types of queries.